webrender/src/renderer.rs

```diff
@@ -119,7 +119,7 @@ pub const MAX_VERTEX_TEXTURE_WIDTH: usize = 1024;
 /// Enabling this toggle would force the GPU cache scattered texture to
 /// be resized every frame, which enables GPU debuggers to see if this
 /// is performed correctly.
-const GPU_CACHE_RESIZE_TEST: bool = false;
+const GPU_CACHE_RESIZE_TEST: bool = true;
```

Comment:
we'll need to flip this back

Reply:
This was just flipped for the sake of testing